### PR TITLE
JN_feature-1-redo

### PR DESCRIPTION
**Pull request by josi543**
feature 1 dispay fields

### CODE REVIEW

File: src/app/features/common-components/registration/registration.component.html [hunk 1]
```diff
@@ -165,6 +165,27 @@ <h4 class="data team-name"> {{ signUpForm.get("teamName").value }} </h4>
...
+            <p class="data">{{ signUpForm.get("developerType").value?.viewValue}}</p>
+          </div>
+
+          <div class="info-group">
+            <p class="category"> Class </p>
+            <p class="data">{{ signUpForm.get("class").value?.viewValue}}</p>
+          </div>
+
+          <div class="info-group">
+            <p class="category"> Years of experience </p>
```
Comment (fxnggg) [inline src/app/features/common-components/registration/registration.component.html]: Nit/suggestion: Capitalize the 'e' in "experience"

**Review by fxnggg — COMMENTED** [review]
### Feature 1 Participant Info Review

Great job! It looks great! I have a couple suggestions, but it's just over small stuff. Aside from the years of experience capitalization comment, I would also suggest changing lines 80 and 82 in the registration.component.html file with "If yes, how many years?" for the previous Hackathon experience to "How many years?" instead and also setting the [ngValue] to "0" rather than null so that the "Select years" is still visible on the raw form. Overall, it looks great! Great work!

File: src/app/features/common-components/registration/registration.component.html (hunk 1 above)
Comment (josi543) [reply src/app/features/common-components/registration/registration.component.html]: Thanks for the input!

**Review by josi543 — COMMENTED** [review]
capitalised e , and changed null to 0

**Review by fxnggg — APPROVED** [review]
Looks great! Great work

**Review by alison-nicole — APPROVED** [review]
Everything looks great!

**Pull request merged**